config/routes.rb

```diff
@@ -114,6 +114,13 @@
   delete '/settings/applications/:application',      to: 'applications#destroy'
   delete '/settings/authorizations/:application',    to: 'authorized_applications#destroy',                      as: :authorization
 
+  # partners is a map of partner special URLs to Stripe coupon IDs
```

Comment:
Putting the partners hash in the routes file seems a bit weird, hard to know where to look for it in the future.  What do you think about putting it in something like an initializer instead?